src/Workspaces/CoreTestUtilities/MEF/UseExportProviderAttribute.cs

```diff
@@ -59,6 +60,14 @@ public class UseExportProviderAttribute : BeforeAfterTestAttribute
 
         private MefHostServices? _hostServices;
 
+        static UseExportProviderAttribute()
+        {
+            // Make sure we run the module initializer for Roslyn.Test.Utilities. C# projects do this via a
+            // build-injected module initializer, but VB projects can ensure initialization occurs by applying the
+            // UseExportProviderAttribute to test classes that rely on it.
+            RuntimeHelpers.RunModuleConstructor(typeof(TestBase).Module.ModuleHandle);
```

Comment:
In the case of c# this will cause the constructor to run twice correct? 

I don't think this impacts correctness really but wanted to make sure I understood the implications.

Reply:
I have no idea. I was expecting semantics similar to `RunClassConstructor`, where it's guaranteed to execute at most once, and exactly once by the time this call returns.

Reply:
Looking at the impl I think it only runs once 

https://github.com/dotnet/runtime/blob/54d3a76eb69143ba59fcf8532ab7600f90161522/src/coreclr/src/vm/reflectioninvocation.cpp#L1796